fission/main.go

```diff
@@ -124,6 +218,35 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "get-env-variables",
```

Comment:
Less than two seconds, depends on where your k8s cluster is.
For apiserver proxy, we still need to know the k8s master ip from the k8s local config. I haven't dig into that.